src/main/reactjs/src/components/contactRequest/ConnectedDialog.tsx

```diff
@@ -35,7 +35,12 @@ export const SuccessDialogWrapper = () => {
       title={t('title')}
       content={<Text>{t('description')}</Text>}
       actions={
-        <Button variant="contained" color="secondary" onClick={() => cleanUp()}>
+        <Button
+          data-testid="success-dialog-continue-btn"
```

Comment:
Pitäisikö data-testid atribuuteissakin käyttää BEM-nimeiskäytäntäjä. Eli silloin yllä oleva id olisi 

success-dialog_continue-btn --> success-dialog__continue-btn

Reply:
Joo, voisihan se olla selkeämpi näin.

Reply:
Muutettu.